README.rst

```diff
@@ -102,13 +102,33 @@ Environment
 
 The *Environment* section is provided by the `pytest-metadata
 <https://pypi.python.org/pypi/pytest-metadata/>`_ plugin, and can be accessed
-via the :code:`pytest_configure` hook:
+via the :code:`pytest_configure` and :code:`pytest_sessionfinish` hooks:
+
+To modify the *Environment* section **before** tests are run, use :code:`pytest_configure`:
 
 .. code-block:: python
 
   def pytest_configure(config):
       config._metadata["foo"] = "bar"
 
+To modify the *Environment* section **after** tests are run, use :code:`pytest_sessionfinish`:
+
+.. code-block:: python
+
+  import pytest
+
+
+  @pytest.hookimpl(tryfirst=True)
+  def pytest_sessionfinish(session, exitstatus):
+      session.config._metadata["Average Response Time"] = calculate_avg_response_time(
+          session
+      )
+
+Note that in the above example `@pytest.hookimpl(tryfirst=True) <https://docs.pytest.org/en/stable/writing_plugins.html#hook-function-ordering-call-example>`_
+is important, as this ensures your :code:`pytest_sessionfinish` method is run **before** any other plugins ( including :code:`pytest-html` and
```

Comment:
Not sure "ensure" is factually correct here (not sure that it matters either 😆 ), but I think pytest doesn't guarantee it, but they will try their best. :)

Reply:
sure, I'll update the wording